public/app/features/plugins/admin/pages/PluginDetails.tsx

```diff
@@ -95,7 +95,7 @@ export default function PluginDetails({ match, queryParams }: Props): JSX.Elemen
         <TabContent className={styles.tabContent}>
           <PluginDetailsSignature plugin={plugin} className={styles.alert} />
           <PluginDetailsDisabledError plugin={plugin} className={styles.alert} />
-          <PluginDetailsBody queryParams={queryParams} plugin={plugin} />
+          <PluginDetailsBody queryParams={queryParams} plugin={plugin} defaultPageId={defaultTab.current} />
```

Comment:
**Question:** couldn't we just pass in the `pageId` here?

Maybe I am wrong, but it looks to me that we are using the same logic in the `PluginDetailsBody` to determine the pageId as what we are doing in this (PluginDetails) component.

Reply:
Thanks for this bro. I've updated it to pass the pageId down. 👍